requirements/docs.txt

```diff
@@ -3,6 +3,7 @@
 
 # These should stay synced with other requirements files
 Babel==2.2.0
+docutils==0.12
```

Comment:
I don't know why this is moved. Just checking that you intended to do this.

Reply:
Yes, I moved it in order to indicate that it is no longer a requirement unique to docs, since it's part of `constraints.txt` (it's needed by `boto3`), and needs to be kept in sync with that file.